src/networking/network.rs

```diff
@@ -299,6 +300,37 @@ impl Network {
             }
         }
     }
+
+    // For transaction đ made by mempool to all peers
+    pub async fn propagate_transaction_to_peers(
+        wallet_lock: Arc<RwLock<Wallet>>,
+        mut tx: Transaction,
+    ) {
+        tokio::spawn(async move {
+            let peers_db_global = PEERS_DB_GLOBAL.clone();
+            let mut peers_db_mut = peers_db_global.write().await;
+            // We need a stream iterator for async(to await send_command_fire_and_forget)
+            let mut peers_iterator_stream = futures::stream::iter(peers_db_mut.values_mut());
+            while let Some(peer) = peers_iterator_stream.next().await {
+                let mut _path = tx.get_path().clone();
```

Comment:
Can we just skip the clone and pass &tx.get_path() directly to is_in_path?

Reply:
Also, why is this 'mut' anyway?? shouldn't the compiler be complaining about that? anyway, I don't think ti's needed at all.